sass/layout/_webhooks.scss

```diff
@@ -108,6 +108,7 @@
             border-left-style: solid;
             border-left-width: 4px;
             padding: 12px;
+            border-left-color: var(--sidebar-header-bg);
```

Comment:
This affects left border color of the detail boxes bots like Jira post:

![Screen Shot 2021-01-26 at 1 20 31 PM](https://user-images.githubusercontent.com/3245614/105887270-8216b400-5fd9-11eb-8852-5d4500fd5aa1.png)